mne/time_frequency/psd.py

```diff
@@ -93,7 +82,7 @@ def _psd_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
     freqs : ndarray, shape (n_freqs,)
         The frequencies.
     """
-    from scipy.signal import welch
+    from scipy.signal import spectrogram
```

Comment:
Then this can just be `spectrogram = get_spectogram()` with a `from ..fixes import get_spectrogram` at the top. See how we do it elsewhere. But wait to see if @agramfort agrees this is a good approach lest we end up bouncing around again :)